lib/ui/painting/image_decoding.cc

```diff
@@ -41,14 +39,18 @@ sk_sp<SkImage> DecodeImage(std::vector<uint8_t> buffer) {
   if (generator == nullptr)
     return nullptr;
 
-  // First, try to create a texture image from the generator.
```

Comment:
I think all of this logic can now be:

// Matt has an upcoming CL that changes this next line to:
// auto srgb = SkColorSpace::MakeSRGB();
auto srgb = SkColorSpace::MakeNamed(SkColorSpace::kSRGB_Named);

auto image = SkImage::MakeFromEncoded(std::move(sk_data));
image = image->makeTextureImage(context, srgb.get());
return image;